custom_components/solarman/translations/it.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Would recommend moving this into a separate PR - would be nice to get it merged in the event that this PR takes a while to be approved.

Reply:
Sure, let me find out how to do that and I'd be happy to have it moved to a separate PR. If you know how to do so and want to give me few pointers, that'd be super welcome. Thanks!

Reply:
@YodaDaCoda , please check Pull Request #243. I took the liberty to separate the addition of some comments on the Readme.MD on #244. Hope that can help. Thanks

Reply:
Can you plz update this PR (make new commits on `main` branch of your fork) to remove this file & changes to the readme as those changes are now covered by #243 & #244?

Reply:
I just re-enabled the commented sensors however I have not found a way to remove the other 2 commits from the branch. Do you have any pointer please? Thanks!

Reply:
Happy to do like I had done for the readme.md, and just revert origin/main to upstream/main, make the change, and then force-push to my origin/main without touching the 2 files referenced in the other PRs. If that's OK just let me know and I'll proceed.

Reply:
you could pull the files from `upstream/main`, commit the changes, then rebase & squash history. more-or-less how i'd do it anyway.

`git fetch origin main`
`git checkout origin/main -- your-file-here`
`git commit ...`
`git rebase -i origin/main`